### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by verissimohenry**
connected the CREATE_BOOK and REMOVE_BOOK actions with the React component and make our page dynamic.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ function App() {
       <h2>BookStore</h2>
       <BooksList />
       <BooksForm />
+      <br />
+      <table>
+        <BooksList />
+      </table>
```
Comment (rindrajosia) [inline src/App.js]: - [x] [REQUIRED] Please make sure that the list of books is displayed only once.

File: src/container/BooksForm.js [hunk 2]
```diff
@@ -1,30 +1,57 @@
...
+      title,
+      category,
+    };
+    props.createBook(book);
+    setTitle('');
+    setCategory('');
+  };
 
   return (
     <form>
```
Comment (rindrajosia) [inline src/container/BooksForm.js:47]: - [x] [REQUIRED]  `handleSubmit` should be attached to the form's `onSubmit` event [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] I got an error related to proptype when I open your APP on my local machine. Please fix it.



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by whiz25 — APPROVED** [review]
## approved 👍🏿👍🏿👍🏿
Nice work making the requested changes. Merge and proceed to the next milestone.


### Martin Afani
#### Slack: whiz25

**Pull request merged**